### ISSUE TITLE
Mac OSX compatibility

### ISSUE THREAD

**Issue by cbess** [cbess/AutoScrollLabel#15]
Allow this to work on the desktop (AppKit).

**Comment by vincentneo** [comment]
Is it actually possible?

**Comment by cbess** [comment]
Absolutely, here is the WIP branch for it: https://github.com/cbess/AutoScrollLabel/tree/mac-support

**Comment by vincentneo** [comment]
But it doesn't seem to work ;(

**Comment by cbess** [comment]
its a "work in progress" (WIP), it may not work
